File: EscolaXavier.Web/src/models/matriculaDto.js
export class MatriculaDto {
    id;
    nomeAluno;
    nomeMateria;
    nota;
  
    constructor(id, nomeAluno, nomeMateria, nota) {
      this.id = id;
      this.nomeAluno = nomeAluno;
      this.nomeMateria = nomeMateria;
      this.nota = nota;
    }
  }

File: Escola.Domain/DTOs/MatriculaDto.cs
namespace Escola.Domain.DTOs
{
    public class MatriculaDto
    {
        public int Id { get; set; }
        public int AlunoId { get; set; }
        public string NomeAluno { get; set; }
        public int MateriaId { get; set; }

        public string NomeMateria { get; set; }
        public decimal? Nota { get; set; }
        public string Status { get; set; }
    }
}
